.github/workflows/CI.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches: [main]
+    tags: ["*"]
+  workflow_dispatch:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
+  cancel-in-progress: true
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCESS_KEY_ID: root
+      AWS_SECRET_ACCESS_KEY: password
```

Comment:
should we add secrets in plain text here, or rather as parameters of the repo? Not sure given that this is just a dummy secret, but ends up being used in the test.